src/Fractions/Formatter/DecimalNotationFormatter.cs

```diff
@@ -72,7 +72,7 @@ public class DecimalNotationFormatter : ICustomFormatter {
     /// <remarks>
     ///     This instance can be used to format Fraction objects into decimal string representations.
     /// </remarks>
-    public static ICustomFormatter Instance { get; } = new DecimalNotationFormatter();
```

Comment:
We need this in order for the docs of the `DecimalNotationFormatter.Instance.Format()` method to display our custom formats (not those of the interface).

Reply:
Ah, right - sorry.